packages/react-ui/src/index.d.ts

```diff
@@ -16,7 +17,7 @@ import {
   TimeSeriesProvider
 } from './widgets/TimeSeriesWidgetUI/hooks/TimeSeriesContext';
 import useTimeSeriesInteractivity from './widgets/TimeSeriesWidgetUI/hooks/useTimeSeriesInteractivity';
-import { CHART_TYPES } from './widgets/utils/constants';
+import { CHART_TYPES } from './widgets/TimeSeriesWidgetUI/utils/constants';
```

Comment:
Exposing types from a specific folder doesn't make a lot of sense to me, what's the reason?

Reply:
CHART_TYPES is a specific enum for the TimeSeriesWidgetUI. It's declared in a file called constants in the utils folder next to another file called utilities. What don't you like from it?